backend/dataall/modules/s3_datasets/services/dataset_profiling_service.py

```diff
@@ -67,13 +67,6 @@ def resolve_profiling_run_status(run_uri):
             session.add(task)
         Worker.queue(engine=context.db_engine, task_ids=[task.taskUri])
 
-    @staticmethod
-    @ResourcePolicyService.has_resource_permission(GET_DATASET)
-    @is_feature_enabled('modules.s3_datasets.features.metrics_data')
-    def list_profiling_runs(uri):
-        with get_context().db_engine.scoped_session() as session:
-            return DatasetProfilingRepository.list_profiling_runs(session, uri)
-
```

Comment:
if removing here - should we not remove `def list_profiling_runs(session, dataset_uri)` from `dataall/backend/dataall/modules/s3_datasets/db/dataset_profiling_repositories.py` ?

Reply:
Done, cleaned up!